src/xrCore/xrDebug.h

```diff
@@ -78,13 +74,27 @@ class XRCORE_API xrDebug
         const char* arg1 = nullptr, const char* arg2 = nullptr);
     static void DoExit(const std::string& message);
 
+	///
+	/// Note: DbgHelp is singlethreaded, so you must synchronize calls to these functions
+	///
+	static void LogStackTrace(const char* header);
+	static xr_vector<xr_string> BuildStackTrace(u16 maxFramesCount = MaxFramesCountDefault);
 private:
     static void FormatLastError(char* buffer, const size_t& bufferSize);
-    static size_t BuildStackTrace(EXCEPTION_POINTERS* exPtrs, char* buffer, size_t capacity, size_t lineCapacity);
     static void SetupExceptionHandler(const bool& dedicated);
     static LONG WINAPI UnhandledFilter(EXCEPTION_POINTERS* exPtrs);
     static void WINAPI PreErrorHandler(INT_PTR);
     static void SaveMiniDump(EXCEPTION_POINTERS* exPtrs);
+
+	///
```

Comment:
One can easily understand that just by looking at the following declarations. Avoid obvious comments - they waste space.